keps/sig-api-machinery/2161-apiserver-default-labels/kep.yaml

```diff
@@ -0,0 +1,29 @@
+title: Immutable label selectors for all namespaces 
+kep-number: 2161
+authors:
+  - "@thockin"
+  - "@jayunit100"
+  - "@rikatz"
+owning-sig: sig-apimachinery
+participating-sigs:
+  - sig-network
+status: provisional
+creation-date: 11-22-2020
+reviewers:
+  - @liggitt
+  - @nikhita
+approvers:
```

Comment:
@liggitt willing to serve as approver?  Since I am too close to it?

Reply:
Oh sorry, changed wrongly. I'll put @liggitt as approver then

Reply:
We should ask him :)

Reply:
Yup Asked in slack he said he'll check it out tomorrow.